app/models/concerns/oregon_digital/ability/collection_ability.rb

```diff
@@ -8,9 +8,18 @@ module CollectionAbility
 
       included do
         def collection_ability
-          can(%i[index show new create edit update delete], Hyrax::CollectionType) if current_user.admin?
+          can(%i[index show new create edit update delete], Hyrax::CollectionType) do collection_type
+            if collection_type.title == "User Collection"
```

Comment:
Two things. I think `collection_type.machine_id == :user_collection` is more robust
and this should simplify to `collection_type.machine_id == :user_collection || current_user.admin?`